### DISCUSSION TITLE
Interfacing to radio

### DISCUSSION THREAD

**Discussion by RobbiePh** [magicbug/Cloudlog#1335] [Q&A] [answered]
New user to Cloudlog and quick question.

So I have both the SATPC32 and Icom Cat add ins installed and configured. Both are Cloudlog Sync and connected.

Using OmniRig 1.20.

On the Cloudlog Dashboard, I don't see any frequencies or modes.

One thing I do notice is that is the add ins connect using http but not https?

Also I only see one radio option in dropdown?

Either way, I am not getting any response in the Cloudlog Dashboard, any thoughts, de Ei2iP

![image](https://user-images.githubusercontent.com/96705124/147423031-d0826ee7-81d7-4b1f-83cf-fcd175005ef1.png)

![image](https://user-images.githubusercontent.com/96705124/147423035-1a6db4ef-7153-41fd-ab55-fd9f0286ab52.png)

![image](https://user-images.githubusercontent.com/96705124/147423042-7a98ea39-eaa9-41c5-8d2e-8ed8949d526d.png)

![image](https://user-images.githubusercontent.com/96705124/147423048-3447f75d-97e0-44d9-b4ad-c30d75a9c8df.png)

![image](https://user-images.githubusercontent.com/96705124/147423081-e2cb84b3-0a7d-4717-9398-0ac225baa811.png)

![image](https://user-images.githubusercontent.com/96705124/147423147-dc1ffeb3-d012-4f5c-b950-51e4f4a39202.png)

**Comment by magicbug** [comment]
SatPC32 looks to be working you just need to select it from the dropdown that you show in the last image.

You can use https or HTTP i'd think it wouldn't matter I'd need to check the omnirig app in the morning

**Reply by RobbiePh** [reply]
SatPC32 Dropdown selected and no data received

![image](https://user-images.githubusercontent.com/96705124/147423698-d341217e-acf0-416e-808b-31885e73a205.png)

![image](https://user-images.githubusercontent.com/96705124/147423700-7e8a456a-3bc5-4773-a1c2-cf3ca88defda.png)

I will play with it a little more.

CloudlogCat will not connect with https

![image](https://user-images.githubusercontent.com/96705124/147423798-b29179a1-8b88-4d99-bba5-a22861b61b4e.png)


![image](https://user-images.githubusercontent.com/96705124/147423823-1883971d-c892-4337-8b8f-30973e90ce9b.png)

**Reply by magicbug** [reply]
If you go to Hardware Interfaces via the dropdown (user name) can you see anything there?

**Comment by magicbug** [comment]
SatPC32 is def working Robbie, just selected it as i'm still logged in from setting it up
![Screenshot from 2021-12-27 00-45-30](https://user-images.githubusercontent.com/84308/147423978-40f2b274-3055-4af3-9016-9767b789c22d.png)

**Reply by RobbiePh** [reply]
Yes now it works, it now seems to connect to the https and I can see the LIVE freq changes. 

Thanks

**Reply by magicbug** [reply]
Woohoo! Okay, I'll test Omnirig in the morning, when i'm not drugged up on sleeping meds lol

**Comment by magicbug** [comment]
![Capture](https://user-images.githubusercontent.com/84308/148134883-a3e48089-5512-42b7-9278-50d48e974b48.PNG)

Sorry for not getting back to you sooner, just tested using https://github.com/magicbug/CloudlogCAT/releases/download/1.3.5/CloudlogCAT.zip

Cloudlog website address with no trailing forward slash on the end and the API key with read+write.

Seems to be working ok

**Comment by magicbug** [comment, marked as answer]
This did have a valid bug and has now been resolved in the code base.
